be/tool/main.c

```diff
@@ -248,6 +254,7 @@ int main(int argc, char *argv[])
 	char                    *path;
 	int                      rc;
 
+	// m0_node_uuid_string_set(NULL);
```

Comment:
Don't use C++ style comments.

If this is not needed, please remove it.

Reply:
Resolved in commit id 5cfda07